models/notications.js

```diff
@@ -0,0 +1,46 @@
+const { Schema, model } = require('mongoose')
+const {
+  enums: { ROLE_ENUM, NOTIFICATION_TYPE }
+} = require('~/consts/validation')
+const { ENUM_CAN_BE_ONE_OF } = require('~/consts/errors')
+
+const { USER, NOTIFICATIONS, REVIEW, COMMENT } = require('~/consts/models')
+
+const notificationsSchema = new Schema(
+  {
+    user: {
+      type: Schema.Types.ObjectId,
+      ref: USER,
+      required: true
+    },
+    userRole: {
+      type: String,
+      enum: {
+        values: ROLE_ENUM,
+        message: ENUM_CAN_BE_ONE_OF('user role', ROLE_ENUM)
+      },
+      required: [true, 'User role must be selected.']
+    },
+    type: {
+      type: String,
+      enum: {
+        values: NOTIFICATION_TYPE,
+        message: ENUM_CAN_BE_ONE_OF('notification type', NOTIFICATION_TYPE)
```

Comment:
changed here to type instead of notification type

Reply:
changed